### DISCUSSION TITLE
Wrong default filename extension when exporting all collections

### DISCUSSION THREAD

**Discussion by dcharalambous** [TechbeeAT/jtxBoard#88] [Q&A] [answered]
Using jtx board on Android 10 (Samsung Galaxy S9+), jtx board version 2.01.03-rc08.gplay (201030008).

While in the Collections menu, when selecting "Export all" from the three-dot-upper-right-menu, the generated zip file has extension .zip.ics

**Comment by patrickunterwegs** [comment]
Hey @dcharalambous !
Thanks for the issue report! Indeed I just reproduced the problem and I'll fix it soon. 
The file is generated correctly, it's only the file ending that is wrong, so as a workaround until it's fixed you can just remove the .ics at the end 👍

**Comment by patrickunterwegs** [comment, marked as answer]
Alright, so the fix is done, it will come with 2.1.5 within the next days 😉

**Comment by dcharalambous** [comment]
Excellent, thanks Patrick!

Best regards,
Demetris

-------- Original Message --------
